out-of-order runbook update with use case (#3253)
grafana/mimir: docs/sources/operators-guide/mimir-runbooks/_index.md

- You are running multiple Prometheus instances pushing the same metrics and [your high-availability tracker is not properly configured for deduplication]({{< relref "../configure/configuring-high-availability-deduplication.md" >}}).
- Prometheus relabelling has been configured and it causes series to clash after the relabelling. Check the error message for information about which series has received a sample out of order.
- A Prometheus instance was restarted, and it pushed all data from its Write-Ahead Log to remote write upon restart, some of which has already been pushed and ingested. This is normal and can be ignored.

> - Prometheus and Mimir has the same recording rule. This will generate the exact same series in both places and will cause either the remote write or the rule evaluation to fail randomly, depending on timing.

Reviewer: ```suggestion
- Prometheus and Mimir have the same recording rule, which generates the exact same series in both places and causes either the remote write or the rule evaluation to fail randomly, depending on timing.
```
